xml/System.Net.Http/WinHttpHandler.xml

```diff
@@ -30,7 +30,7 @@
   
 ## Remarks
 
- <xref:System.Net.Http.WinHttpHandler> is similar to other existing classes such as <xref:System.Net.Http.HttpClientHandler>. <xref:System.Net.Http.WinHttpHandler> provides a handler underneath an <xref:System.Net.Http.HttpClient> instance and is used to send HTTP requests out to a server and receive server responses.  
+ <xref:System.Net.Http.WinHttpHandler> is similar to other existing classes such as <xref:System.Net.Http.HttpClientHandler>. <xref:System.Net.Http.WinHttpHandler> provides a handler underneath an <xref:System.Net.Http.HttpClient> instance and is used to send HTTP requests out to a server and receive server responses. The <xref:System.Net.Http.WinHttpHandler> implementation supports HTTP versions up to HTTP/2.  
```

Comment:
Would we throw for H/3? Or any different version from H/1(.1) and H/2? Just wondering what the actual behavior is.

Reply:
I don't see any validation logic. I think we would silently treat HTTP/3 (or any other unknown) requests as if they were HTTP/1(.1).

Reply:
Anton is right, we pass null to WinHTTP and it uses H/1.1 as the default.